e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -238,6 +238,54 @@ class RegistrationDetails {
     ).toContain(userName);
     expect(await timeStampLocator.textContent()).toContain(date);
   }
+
+  async addNote(actions: string, addNote: string) {
+    await this.page.getByText(actions).click();
+    await this.page.getByText(addNote).click();
+  }
+
+  async writeNote(placeholder: string, note: string, buttonName: string) {
```

Comment:
This is not only true here, but true in our whole codebase (and a personal gripe I have in many places in the 121-service). When using multiple arguments like this, it becomes hard to understand what is going on in the test itself. I look at where `writeNote` is being used, and I have no idea what is being passed into the function.

I would recommend using [an options object](https://stackoverflow.com/a/12827204) instead of multiple parameters, which besides readability has several other advantages (for example, order of arguments becomes irrelevant).

For example:

```suggestion
  async writeNote({
    placeholder,
    note,
    buttonName
  }: {
    placeholder: string,
    note: string,
    buttonName: string
  }) {
```

I will also add a suggestion below (where this is called) to demonstrate the increase in readability and I would recommend doing this for other new functions for consistency.

Reply:
Okay this is gold in here